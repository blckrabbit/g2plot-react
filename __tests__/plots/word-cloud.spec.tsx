import React from 'react'
import { create } from 'react-test-renderer'
import WordCloudChart from '../../src/plots/word-cloud'

describe('WordCloudChart', () => {
  test('should render without crashed', () => {
    const renderer = create(
      <WordCloudChart data={[]} tooltip={{ visible: true }} />
    )

    expect(renderer.toJSON()).toMatchSnapshot()
  })
})
