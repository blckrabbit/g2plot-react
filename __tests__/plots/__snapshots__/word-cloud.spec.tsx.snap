// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WordCloudChart should render without crashed 1`] = `<div />`;
